src/components/full-calender/components/shared/register-form/reducer/password.js

```diff
@@ -0,0 +1,13 @@
+const initialState = {
+    error: true,
+    value: ""
+};
+
+export default (state = initialState, action) => {
+    switch (action.type) {
+        case "PASSWORD_AVAILABILITY": return action.payload;
+        case "RESET_PASSWORD_DATA": return initialState;
```

Comment:
it should be ...
        case "RESET_PASSWORD_DATA":
        case "RESET_REGISTER_FORM":
                  return initialState;